source/features/update-pr-from-base-branch.tsx

```diff
@@ -11,8 +13,11 @@ import {getBranches} from '../github-helpers/pr-branches';
 import getPrInfo from '../github-helpers/get-pr-info';
 import {getConversationNumber} from '../github-helpers';
 import showToast from '../github-helpers/toast';
+import createMergeabilityRow from '../github-widgets/mergeability-row';
+import selectHas from '../helpers/select-has';
 
-const selectorForPushablePRNotice = '.merge-pr > .color-fg-muted:first-child';
+const canMerge = '.merge-pr > .color-fg-muted:first-child';
```

Comment:
- https://github.com/refined-github/refined-github/issues/6334

I'm not seeing it in our own repo though 🤷‍♂️